### PR TITLE
Cld 306 pwa pgw swatch issue

### PR DESCRIPTION
**Pull request by asad-rafter**

### CODE REVIEW

File: Cloudinary-pwa-kit/packages/pwa-cloudinary/app/components/cloudinary-widgets/index.jsx [hunk 1]
```diff
@@ -4,22 +4,26 @@ import { cloudinary } from '../../../config/default'
...
-                } else {
-                    window.cldGallery = window.cloudinary.galleryWidget(galleryOptions)
-                }
+    const renderGallery = async () => {
+        const gallery = document.querySelector('#cld-gallery')
+        const galleryOptions = cloudinaryImageGallery.cloudinaryImage.galleryWidget.options
+        galleryOptions.queryParam = cloudinary.CLD_PGW_TRACKING_PARAM
+        if (typeof window !== 'undefined' && window.cloudinary && window.cloudinary.galleryWidget) {
+            if (window.cldGallery) {
+                gallery.style.visibility = 'hidden'
```
Comment (asad-rafter) [inline Cloudinary-pwa-kit/packages/pwa-cloudinary/app/components/cloudinary-widgets/index.jsx]: @yuval-cloudinary, I just tested, if we remove this `gallery.style.visibility = 'hidden'` and `gallery.style.visibility = 'visible'` it works fine, after our changes of `await cldGallery.update(galleryOptions)`

File: Cloudinary-pwa-kit/packages/pwa-cloudinary/app/components/cloudinary-widgets/index.jsx [hunk 2]
```diff
@@ -4,22 +4,26 @@ import { cloudinary } from '../../../config/default'
...
+                await cldGallery.update(galleryOptions)
```
Comment (yuval-cloudinary) [inline Cloudinary-pwa-kit/packages/pwa-cloudinary/app/components/cloudinary-widgets/index.jsx]: for consistency, I would use window.cldGallery

File: Cloudinary-pwa-kit/packages/pwa-cloudinary/app/components/cloudinary-widgets/index.jsx [hunk 3]
```diff
@@ -4,22 +4,26 @@ import { cloudinary } from '../../../config/default'
...
+                gallery.style.visibility = 'visible'
+            } else {
+                window.cldGallery = window.cloudinary.galleryWidget(galleryOptions)
                 cldGallery.render()
```
Comment (yuval-cloudinary) [inline Cloudinary-pwa-kit/packages/pwa-cloudinary/app/components/cloudinary-widgets/index.jsx]: for consistency, I would use window.cldGallery

**Review by yuval-cloudinary — CHANGES_REQUESTED** [review]
Great news. A couple of styling change requests

File: Cloudinary-pwa-kit/packages/pwa-cloudinary/app/components/cloudinary-widgets/index.jsx (hunk 1 above)
Comment (asad-rafter) [reply Cloudinary-pwa-kit/packages/pwa-cloudinary/app/components/cloudinary-widgets/index.jsx]: do you want me to remove the visibility attribute or stick with it?

Comment (yuval-cloudinary) [reply Cloudinary-pwa-kit/packages/pwa-cloudinary/app/components/cloudinary-widgets/index.jsx]: @asad-rafter yes, please remove the visibility attribute. That would be the last change and I'll close and merge it

File: Cloudinary-pwa-kit/packages/pwa-cloudinary/app/components/cloudinary-widgets/index.jsx (hunk 3 above)
Comment (yuval-cloudinary) [reply Cloudinary-pwa-kit/packages/pwa-cloudinary/app/components/cloudinary-widgets/index.jsx]: ^

File: Cloudinary-pwa-kit/packages/pwa-cloudinary/app/components/cloudinary-widgets/index.jsx (hunk 2 above)
Comment (yuval-cloudinary) [reply Cloudinary-pwa-kit/packages/pwa-cloudinary/app/components/cloudinary-widgets/index.jsx]: ^

File: Cloudinary-pwa-kit/packages/pwa-cloudinary/app/components/cloudinary-widgets/index.jsx (hunk 3 above)
Comment (asad-rafter) [reply Cloudinary-pwa-kit/packages/pwa-cloudinary/app/components/cloudinary-widgets/index.jsx]: apologies for this miss, I just updated

**Pull request merged**